Parsers/ASim WindowsEvent/ARM/MicrosoftWindowsEventFullDeployment.json

```diff
@@ -82,10 +82,10 @@
                     "contentVersion": "1.0.0.0"
```

Comment:
Might be a review issue, but I think that vimAuthentication is not included. 

Also, unrelated to this PR, but ASimNetwork might be missing.

Reply:
vimAuthentication - duplicate comment. I replied on the other thread.